profiles/govcms_saas.yml

```diff
@@ -88,3 +88,23 @@ checks :
     max_size : 5
     warning_size : 2
   \SiteAudit\Check\Phantomas\InPage404s : {}
+
+  # Field specific checks.
+  \SiteAudit\Check\D7\EntityReferenceAutocomplete:
+    threshold: 100
+
+  # Module configuration checks.
+  \SiteAudit\Check\D7\ModuleSettings:
```

Comment:
Is it possible to have multiple `ModuleSettings` checks declared in the same profile?